browser/ui/side_panel/ai_chat/ai_chat_side_panel_utils.cc

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "brave/browser/ui/side_panel/ai_chat/ai_chat_side_panel_utils.h"
+
+#include "chrome/browser/ui/browser_finder.h"
+
+namespace ai_chat {
+
+#if !defined(TOOLKIT_VIEWS)
```

Comment:
also why is this here at all if it's NOTIMPLEMENTED?

Reply:
Looks like because of https://github.com/brave/brave-core/pull/17402/files/26517085fe2257477c9e684ace103c42efbf1000#r1157861778. I'm not sure why it's needed in the current implementation, but I suppose it can actually get a Browser if the code path is ever used and not use NOTIMPLEMENTED...